NEWS.md

```diff
@@ -3,6 +3,7 @@
 # pkgdown 1.4.1
 
 * Don't install test package in user library (fixes CRAN failure).
+* `deploy_site_github()` now accepts a `host` argument enabling users to leverage this for github enterprise - @dimagor
```

Comment:
The auto-linking requires the attribution be in parentheses. Please update to `(@dimagor)`.